irm/google/cloud/irm_v1alpha2/gapic/enums.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Wrappers for protocol buffer enum types."""
+
+import enum
+
+
+class Artifact(object):
+    class Type(enum.IntEnum):
+        """
+        Possible types of an artifact.
+
+        Attributes:
+          TYPE_UNSPECIFIED (int): External type is unspecified.
+          TYPE_URL (int): URL.
+          TYPE_ISSUE (int): An issue.
+          TYPE_SOURCE_CONTROL_CHANGE (int): A source control change.
+          TYPE_JIRA_ISSUE (int): A JIRA issue.
```

Comment:
This seems odd to me. we have `issue` and `jira_issue`. What is `issue` representing. Is there a default issue db?

Reply:
@busunkim96 The GAPIC config for IRM has been [updated to include scopes](https://github.com/googleapis/googleapis/commit/84f9bf2c274694a632b67ca0efcad20c56ead8cd).